ome_model/experimental.py

```diff
@@ -105,6 +105,10 @@ def __init__(self,
                  tiffs=[],
                  order="XYZTC",
                  type="uint16",
+                 physSizeX="1.0", physSizeY="1.0", physSizeZ="1.0",
```

Comment:
Primary question is about the extensibility and the consistency of the API as we want to make more optional properties available to `Image` but also other objects. The two options I have been debating previously are:

- support for each new attribute as a key/value pair as done here
- passing an optional dictionary e.g. https://github.com/ome/ome-model/blob/38624e963f4dbde67992416d000d9eef26828f3b/test/unit/test_image.py#L254-L267. 

My preference goes for the second form as it enforces the name of the attributes that can be passed to the optional dictionary to match the XSD schema.